LEVEL5/2562.py

```diff
@@ -0,0 +1,7 @@
+aList = []
+
+for i in range(9):
+    aList.append(int(input())) #input 9번 생성 후 삽입
```

Comment:
관례상 반복문 변수가 의미가 없을 때는 underscore 를 사용합니다
```suggestion
for _ in range(9):
    aList.append(int(input())) #input 9번 생성 후 삽입
```

Reply:
그리고 리스트 컴프리핸션이 가능하죠
```python
aList = [int(input()) for _ in range(9)]
```